frog/imports/client/GraphEditor/Graph.js

```diff
@@ -23,7 +23,6 @@ const scrollMouse = e => {
 };
 
 const mousemove = e => {
-  // We do -100 here because there is 100px above the graph editor
```

Comment:
I am not sure if we should remove this comment, it provides an explanation for the magic number below.